### PR TITLE
20046 - Convert components to composition-api

### PR DESCRIPTION
**Pull request by sameer0422**
*Issue #:*
https://github.com/bcgov/entity/issues/20046

*Description of changes:* Covert components to composition-api

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2946-nnduf015.web.app

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 1]
```diff
@@ -291,404 +291,375 @@
...
+      const accountSettings = getAccountFromSession()
+      console.log("accountSettings", accountSettings)
+      await orgStore.syncOrganization(accountSettings?.id)
+      setAccountChangedHandler(setup)
+      await setup()
+    })
+
+    onBeforeUnmount(() => { beforeDestroy() })
+
+    return {
```
[the remark this answers is not in the document]
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:601]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2946-nnduf015.web.app

**Review by seeker25 — DISMISSED** [review]

**Comment by Jxio** [comment]
it sends empty payload when I modify mailing address:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2946-nnduf015.web.app

**Comment by sameer0422** [comment]
@Jxio @seeker25  please recheck this PR.
I fixed all the issues.

**Review by avni-work — APPROVED** [review]

**Review by Jxio — APPROVED** [review]

**Pull request merged**